test/behaviors/UnstructuredUpgradeableToken.js

```diff
@@ -86,20 +84,6 @@ const shouldBehaveLikeUnstructuredUpgradeableTokenV0 = (
     );
   });
 
-  context(
```

Comment:
So one issue with this is that unstructuredtontokentedts actually was testing almost all of the 777 functionalities before. Removing this drastically reduces code coverage. If we want to remove this we should rewrite the test cases existing inside of here.